client/services.go

```diff
@@ -598,3 +599,12 @@ type WorkspacesClient interface {
 	DescribeWorkspaces(ctx context.Context, params *workspaces.DescribeWorkspacesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspacesOutput, error)
 	DescribeWorkspaceDirectories(ctx context.Context, params *workspaces.DescribeWorkspaceDirectoriesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspaceDirectoriesOutput, error)
 }
+
+//go:generate mockgen -package=mocks -destination=./mocks/backup.go . BackupClient
+type BackupClient interface {
```

Comment:
This is a global table API https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeGlobalSettings.html
https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeRegionSettings.html

Reply:
Would open an issue for this as well.

Reply:
https://github.com/cloudquery/cq-provider-aws/issues/674

Reply:
https://github.com/cloudquery/cq-provider-aws/issues/674